packages/forklift-console-plugin/src/modules/Providers/utils/validators/provider/openshift/openshiftSecretFieldValidator.ts

```diff
@@ -25,7 +28,7 @@ export const openshiftSecretFieldValidator = (id: string, value: string): Valida
       validationMsg = validateToken(trimmedValue);
       break;
     case 'insecureSkipVerify':
-      validationMsg = { type: 'default', msg: 'Migrate without validating a CA certificate' };
+      validationMsg = { type: 'default', msg: 'Skip certificate validation' };
```

Comment:
nitpick:
```suggestion
      validationMsg = { type: 'default', msg: 'Skip CA certificate validation' };
```

Reply:
This is not relevant anymore after adding `validateInsecureSkipVerify`